components/Button/Button.css

```diff
@@ -1,3 +1,6 @@
 .Button {
```

Comment:
This is a small suggestion for possible future projects! You can change the cursor to be a pointer when it hovers over the links/buttons.